wrappers/javascript/anoncreds-shared/src/api/RevocationRegistry.ts

```diff
@@ -4,12 +4,6 @@ import type { RevocationRegistryDefinition } from './RevocationRegistryDefinitio
 import { AnoncredsObject } from '../AnoncredsObject'
 import { anoncreds } from '../register'
 
-export type RevokeCredentialOptions = {
```

Comment:
Shouldn't the `tailPath` field just be removed?

Reply:
I checked and this structure isn't used anywhere

Reply:
Okay, then it is all good.